### PR TITLE
Financial docs v1.1

### PR DESCRIPTION
**Pull request by ianardee**
## Description
Autogenerated support for financial docs v1.1

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires a change to the official Guide documentation.

### CODE REVIEW

File: src/documents/financialDocument/financialDocumentV1LineItem.ts [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+  quantity: number | null;
+  /** The item tax amount. */
+  taxAmount: number | null;
+  /** The item tax rate in percentage. */
+  taxRate: number | null;
+  /** The item total amount. */
+  totalAmount: number | null;
+  /** The item unit price. */
+  unitPrice: number | null;
+
```
Comment (sebastianMindee) [inline src/documents/financialDocument/financialDocumentV1LineItem.ts:22]:
```suggestion
pageId: number;

```

File: src/documents/financialDocument/financialDocumentV1LineItem.ts [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+      this.taxRate = null;
+    }
+    this.totalAmount = +parseFloat(prediction["total_amount"]).toFixed(3);
+    if (isNaN(this.totalAmount)) {
+      this.totalAmount = null;
+    }
+    this.unitPrice = +parseFloat(prediction["unit_price"]).toFixed(3);
+    if (isNaN(this.unitPrice)) {
+      this.unitPrice = null;
+    }
```
Comment (sebastianMindee) [inline src/documents/financialDocument/financialDocumentV1LineItem.ts:45]:
```suggestion
    }
    this.pageId = prediction.page_id;
```

File: src/documents/receipt/receiptV5LineItem.ts
Comment (sebastianMindee) [inline src/documents/receipt/receiptV5LineItem.ts:1]: `pageId` seems to be missing here as well?

**Review by sebastianMindee — COMMENTED** [review]
`pageId` field seems to be missing on `ReceiptV5LineItem` as well as `FinancialDocumentV1LineItem` despite being present on others as well as on their equivalent on the Python SDK.

Also: `confidence` score seems to be missing as well, though this one is also absent on the Python SDK.

File: src/documents/receipt/receiptV5LineItem.ts
Comment (ianardee) [reply src/documents/receipt/receiptV5LineItem.ts:1]: This is a problem with the code generator, will fix.

**Review by sebastianMindee — APPROVED** [review]
:+1:

**Pull request merged**